language/sentiment/README.txt

```diff
@@ -0,0 +1,27 @@
+# Introduction
+
+This sample contains the code referenced in the 
+[Sentiment Analysis Tutorial](http://cloud.google.com/natural-language/docs/sentiment-tutorial)
+within the Natural Language API Documentation. A full walkthrough of this sample
+is located within the documentation.
+
+This sample is a simple illustration of how to construct a sentiment analysis
+request and process a response using the API.
+
+## Download the Code
+
+```
+git clone sso://python-dev-samples/language/sentiment/
+```
+
+### Usage
```

Comment:
Yes, every sample should use a virtualenv. Not using a virtualenv is like going out in the rain without an umbrella.